buildconfigs/30-machine-os-content.yaml

```diff
@@ -7,41 +7,24 @@ spec:
   source:
     type: Git
     git:
-      uri: 'https://github.com/vrutkovs/okd-machine-os'
-      ref: master-coreos-layering
+      uri: 'https://github.com/openshift/okd-machine-os'
+      ref: master # TODO: Change to release-4.12
     contextDir: .
     images:
       - from:
           kind: ImageStreamTag
-          name: 'release:hyperkube-rpms'
+          name: 'release:artifacts'
         as:
-          - 'registry.ci.openshift.org/origin/4.12:hyperkube-rpms'
+          - 'registry.ci.openshift.org/origin/4.12:okd-rpms'
```

Comment:
Shouldn't this be `registry.ci.openshift.org/origin/4.12:artifacts`?

Reply:
yes, let me verify even the rest. Thanks